Source/WebCore/platform/SharedBuffer.cpp

```diff
@@ -35,6 +35,17 @@
 #include <wtf/text/StringBuilder.h>
 #include <wtf/unicode/UTF8Conversion.h>
 
+static constexpr size_t minimumPageSize = 4096;
+
+constexpr bool useUnixDomainSockets()
```

Comment:
Maybe this can be a static global constexpr boolean variable instead. Not much point in have a function here I think (I know you just copied the code over).